### DISCUSSION TITLE
Support for SFTP for Azure Blob Storage w/in the SFTP-SSH managed connector

### DISCUSSION THREAD

**Discussion by gregorywilliamwalker** [Azure/logicapps#706] [Feature Ideas/Requests]
Our org is moving away from a vm-based sftp server to using the SFTP for Azure Blob Storage. We have some Logic Apps that will need access to data on the SFTP for Azure Blob Storage in order to complete this. It would be great to get this supported w/in the SFTP-SSH managed connector asap.

**Comment by karl-lonnberg** [comment]
Hi

I'm just about to build this as well. Is there any problems accessing the data using a regular storage/blob connector?
I imagined this would work with SFTP-SSH at least, what is the problem there?
